### PR TITLE
Fixes PATH variance, filename replacement script

### LINKED ISSUE #10662: Resolve difference between install and PATH instructions for Linux
### Page URL

https://docs.flutter.dev/get-started/install/linux/desktop

### Page source

_No response_

### Describe the problem

["Download then install flutter - Step 3"](https://docs.flutter.dev/get-started/install/linux/desktop?tab=download#download-then-install-flutter)
> Extract the file into the directory you want to store the Flutter SDK.
> `tar -xf ~/Downloads/flutter_linux_3.22.1-stable.tar.xz -C /usr/bin/`
> When finished, the Flutter SDK should be in the /usr/bin/flutter directory.

The next step:
["Add Flutter to your PATH"](https://docs.flutter.dev/get-started/install/linux/desktop?tab=download#add-flutter-to-your-path)
> To run Flutter commands in a shell, add Flutter to the PATH environment variable.
> ...
> To add Flutter to your PATH, expand the entry for your default shell, then choose the command.
> `echo 'export PATH="$HOME/development/flutter/bin:$PATH"' >> ~/.bash_profile`

### Expected fix

Update the instructions to use the same installation location.

### Additional context

_No response_

### I would like to fix this problem.

- [X] I will try and fix this problem on docs.flutter.dev.

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #10662

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/_data/shells.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 - name: bash
```
Comment (sfshaza2) [inline src/_data/shells.yml:1]: Might it be that Flutter is NOT in /usr/bin?

File: src/_includes/docs/install/flutter/download.md [hunk 2]
```diff
@@ -8,17 +8,17 @@
    {% assign path='C:\\user\\{username}\\dev' %}
    {% assign flutter-path='C:\\user\\{username}\\dev\\flutter' %}
    {% assign terminal='PowerShell' %}
-   {% assign prompt='C:>' %}
+   {% assign prompt='PS C:\\>' %}
```
Comment (sfshaza2) [inline src/_includes/docs/install/flutter/download.md:11]: I thought we weren't using :PS C"?

**Review by sfshaza2 — COMMENTED** [review]
Not a full tech review, just questions.

File: src/_includes/docs/install/flutter/download.md (hunk 2 above)
Comment (atsansone) [reply src/_includes/docs/install/flutter/download.md:11]: We encourage PowerShell everywhere, and the PowerShell prompt is `PS C:\\>`. I noticed it on my PC the other day and wanted to be consistent / correct.

File: src/_data/shells.yml (hunk 1 above)
Comment (atsansone) [reply src/_data/shells.yml:1]: True, but this is consistent with the previous advice given on the page, which is to consider adding `flutter` to `/usr/bin`. If they follow inline, this should be OK.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**